docs/clients/dovecot.txt

```diff
@@ -45,6 +45,28 @@ Configuring Dovecot to work with Virtual Exim
    Dovecot expects the select to return a set of columns with particular
    names, hence usage of the 'AS' keyword above.
 
-3. There's no step 3! Just restart Dovecot, and it should work.
+3. Exim can’t directly handle SHA512-crypt for user authentication but exim can use
+   the dovecot authenticator to login that we need to enable.
+   For more information: http://wiki2.dovecot.org/HowTo/EximAndDovecotSASL
+
+   Edit /etc/dovecot/conf.d/10-master.conf and uncomment these lines
+
+     service auth {
+     ...
+     #SASL
+       unix_listener auth-client {
+       mode = 0660
+       user = vexim
+       }
+     ...
+     }
+
+   and edit /etc/dovecot/conf.d/10-auth.conf
+     auth_mechanisms = plain login
+
+4. Edit your /etc/exim4/exim4.conf or /etc/exim4/conf.d/auth/30_vexim_authenticators
+   and uncomment the dovecot-authenticator section. 
+
+5. There's no step 3! Just restart Dovecot and Exim, and it should work.
```

Comment:
There's no step 5, not 3. :)
Also, I think you should add yourself to the credits below, or remove them altogether.
Although, if we move our documentation to the Wiki, this file will probably become unnecessary altogether.